cpp/oneapi/dal/algo/logistic_regression/test/fixture.hpp

```diff
@@ -198,14 +248,23 @@ class log_reg_test : public te::crtp_algo_fixture<TestType, Derived> {
     double C_ = 1.0;
     std::int64_t n_ = 0;
     std::int64_t p_ = 0;
-    array<float_t> X_host_;
+    std::int64_t train_size_ = 0;
+    std::int64_t test_size_ = 0;
+    array<float_t> X_train_host_;
+    array<float_t> X_test_host_;
     array<float_t> params_host_;
-    array<std::int32_t> y_host_;
-    array<std::int32_t> resp_;
+    array<std::int32_t> y_train_host_;
+    array<std::int32_t> y_test_host_;
+
+    array<std::int32_t> resp_train_;
+    array<std::int32_t> resp_test_;
+    table X_train_;
+    table X_test_;
 };
 
 using log_reg_types = COMBINE_TYPES((float, double),
-                                    (logistic_regression::method::dense_batch),
+                                    (logistic_regression::method::dense_batch,
+                                     logistic_regression::method::sparse),
                                     (logistic_regression::task::classification));
```

Comment:
may be its not necessary to add `logistic_regression::task::classification` due to you have only this task. The same comment about spmd types

Reply:
I think it is fine to keep it like that, this gives the sense of what template parameters are used in the algorithm